### DISCUSSION TITLE
Suggestion: Add some additional info for Linux & macOS users in the readme.md

### DISCUSSION THREAD

**Discussion by wilsontulus** [ThioJoe/YT-Spammer-Purge#64] [Ideas]
For some reason, I just thought that while the raw .py file can be used in almost every platforms, there are probably some Linux / macOS users might think that this is just a Windows program and I don't want them running the .exe using WINE or something, as running a Python script with WINE is very inefficient.

So, I suggest that some notes should be added for Linux / macOS users at the readme so they will be encouraged to use the YoutubeSpammerPurge.py script. Especially that some YouTubers are using Macs, or even Linux, as their daily-driver.

I could just make a pull request & edit the README.md, but I don't want my bad English grammar to ruin it.
Thanks, from @wilsontulus

**Comment by reesericci** [comment]
AppImage/Flatpak for Linux users would be nice. Or maybe a self-contained binary.

**Comment by Martinligabue** [comment]
is this helpful?
https://github.com/niess/python-appimage

**Comment by wilsontulus** [comment]
No, AppImage / Flatpak isn't really a nice solution though as most Linux distributions already come with Python installed (even though most of them didn't came with required prerequisites installed).
What I meant was, I want the download instructions at the readme to tell Linux & macOS users to install Python in their systems (if Python haven't installed yet), then install the required prerequisites, and finally they can run the program by executing the **YouTubeSpammerPurge.py** script. 
In conclusion, just a Windows instruction isn't enough, as there are also many YouTubers using macOS as their daily driver, and even some (mostly tech) YouTubers are using Linux as their daily driver.

I could just make a pull request & edit the README.md, but I don't want my bad English grammar to ruin it.

Thanks, sorry for my grammar though 😊

**Comment by reesericci** [comment]
@wilsontulus self-contained binary?

**Comment by Martinligabue** [comment]
@wilsontulus I wouldn't care about grammar, someone else will correct it if it's weird, but I think it would be at least a first step to have a draft
I also can help if there's need

**Comment by ThioJoe** [comment]
Added Linux instructions: https://github.com/ThioJoe/YouTube-Spammer-Purge/wiki/Linux-Installation-Instructions
Also MacOS instructions: https://github.com/ThioJoe/YouTube-Spammer-Purge/wiki/MacOS-Instructions

Added links to the readme. I'll convert this to a discussion in case there are any additional thoughts.
